### PR TITLE
Operator integrations for node_exporter and mysqld_exporter

### PR DESCRIPTION
**Pull request by karengermond**
#### Which issue(s) this PR fixes

Closes https://github.com/grafana/agent/issues/2178

#### PR Checklist

- [ ] CHANGELOG updated
- [x] Documentation added
- [ ] Tests updated

### CODE REVIEW

**Review by jcreixell — APPROVED** [review]
I have only verified the node_exporter one in my local env, but assuming that the mysqld one works, LGTM!

File: docs/sources/operator/operator-integrations.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+---
+# Set up Agent Operator integrations
+
+This topic provides examples of setting up Agent Operator integrations, including [node_exporter](#set-up-an-agent-operator-node_exporter-integration) and [mysqld_exporter](#set-up-an-agent-operator-mysqld_exporter-integration).
+
+## Before you begin
+
+Before you begin, make sure that you have deployed the Grafana Agent Operator CRDs and installed Agent Operator into your cluster. See [Install Grafana Agent Operator with Helm]({{< relref "./helm-getting-started.md" >}}) or [Install Grafana Agent Operator]({{< relref "./getting-started.md" >}}) for instructions.
+
+Also, make sure that you [deploy the GrafanaAgent resource]({{< relref "./deploy-agent-operator-resources.md" >}}) and the `yaml` you use has the `integrations` definition under `spec`.
```
Comment (karengermond) [inline docs/sources/operator/operator-integrations.md:13]: @marctc , does this information belong here: "the `yaml` you use has the `integrations` definition under `spec`?"  Should we add this within the steps?

File: docs/sources/operator/operator-integrations.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+           path: /proc
+       - name: sys
+         hostPath:
+           path: /sys
+       - name: root
+         hostPath:
+           path: /root
+    ```
+
+    > **Important:** In order to make integrations work, the field `name` under the `spec` section of the manifest must contain the name of the integration to be installed according to the list of integrations defined [here]({{< relref "../configuration/integrations/integrations-next/_index.md#config-changes" >}}).
```
Comment (karengermond) [inline docs/sources/operator/operator-integrations.md:63]: @marctc I moved this paragraph from the overview to within the steps and made it an "important" note.  I think that works better to include the information right when they need it.

File: docs/sources/operator/operator-integrations.md (hunk 1 above)
Comment (marctc) [reply docs/sources/operator/operator-integrations.md:13]: I assume that users have followed previous steps (before you begin). This is just a reminder for that

**Review by jcreixell — APPROVED** [review]
added a suggestion, thanks for adding the clarifications @marctc, if you are happy with it let's :ship:

**Pull request merged**